### PR TITLE
Print no pairings message when checking a client with no pairs

### LINKED ISSUE #183: [PE-D][Tester B] Program exits itself when listing clients with no paired homes
Adding a client without pairing him/her with a house. Checking the client will cause the program to throw an exception and crash.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: HT-T/ped#10

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**
Fixes #183

### CODE REVIEW

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -283,6 +284,12 @@ public void showCheckClient(Client client, Property property) {
         showToUser("  " + property.toString());
     }
 
+    public void showCheckClientNoPair(Client client) {
+        showToUser(MESSAGE_CHECK_CLIENT);
+        showToUser("  " + client.toString() + "\n");
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:289]: Would be good to avoid magic literals.

File: src/main/java/seedu/duke/Messages.java [hunk 2]
```diff
@@ -186,6 +186,8 @@ public class Messages {
 
     public static final String MESSAGE_CHECK_CLIENT_RESULT = "Here is the property this client is renting:";
 
+    public static final String MESSAGE_CHECK_CLIENT_NO_PAIR = "There are no pairings involving this client.";
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Messages.java:189]: I really like how clear your error message is.

**Review by wilsonngja — COMMENTED** [review]
Good job in fixing the errors. LGTM

**Review by ngdeqi — APPROVED** [review]
Follows code quality guidelines. LGTM

**Pull request merged**